master/internal/api_tasks.go

```diff
@@ -341,7 +345,14 @@ func (a *apiServer) GetTaskAcceleratorData(
 
 	var accelerationData []*apiv1.AcceleratorData
 	for _, r := range res {
-		accelerationData = append(accelerationData, r.Proto())
+		accelerationData = append(accelerationData, &apiv1.AcceleratorData{
```

Comment:
why not modify the Proto() method?

Reply:
I'm hesitate to change the `model.AcceleratorData` type, it will no longer match the `allocation_accelerators` table, also there are use cases where the `resource_pool` field is not necessary